ftw/participation/tests/test_reminder.py

```diff
@@ -0,0 +1,61 @@
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.participation.interfaces import IParticipationSupport
+from ftw.participation.tests.layer import FTW_PARTICIPATION_FUNCTIONAL_TESTING
+from ftw.testbrowser import browsing
+from ftw.testing.mailing import Mailing
+from plone.app.testing import login
+from plone.app.testing import TEST_USER_ID
+from plone.app.testing import setRoles
+from plone import api
+from unittest import TestCase
+import transaction
+
+
+class TestReminder(TestCase):
+
+    layer = FTW_PARTICIPATION_FUNCTIONAL_TESTING
+
+    def setUp(self):
+
+        self.portal = self.layer['portal']
+        Mailing(self.portal).set_up()
+        transaction.commit()
+
+
+
+        self.folder = create(Builder('folder')
+                             .titled(u'The F\xf6lder')
+                             .providing(IParticipationSupport))
+
+        self.inviter = api.user.get(userid=TEST_USER_ID)
+        self.grant('Manager')
+
+    def grant(self, *roles):
+        setRoles(self.portal, TEST_USER_ID, list(roles))
+        transaction.commit()
+
+    def tearDown(self):
+        Mailing(self.layer['portal']).tear_down()
+
+    @browsing
+    def test_send_reminder_mail(self, browser):
+
+        matthias = create(Builder('user')
```

Comment:
Could you please remove me from the code here 🧐. Also, it would be great if the user had some umlauts in his/her name.

Reply:
I don't think that umlaute are really common in E-Mail adresses.